src/ToggleBadge.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { theme as themeGet, space } from 'styled-system'
+import theme from './theme'
+
+const ToggleBadge = styled.div`
+  border-radius: 2px;
```

Comment:
You may want to use the border-radius from theme, instead of hardcoding it, even though it is the same value.  See `Card` as an example.